pkgs/top-level/all-packages.nix

```diff
@@ -3703,6 +3703,8 @@ with pkgs;
 
   dconf = callPackage ../development/libraries/dconf { };
 
+  dconf-utils = callPackage ../development/libraries/dconf/utils.nix { };
```

Comment:
This is very tiny, please move it into dconf itself.

Reply:
It is only tiny because it is very incomplete. We will want something like https://github.com/nix-community/home-manager/blob/ec06f419af79207b33d797064dfb3fc9dbe1df4a/modules/lib/gvariant.nix